src/modules/content-sqlite/content-sqlite.c

```diff
@@ -807,7 +807,7 @@ static int process_args (struct content_sqlite *ctx,
             *truncate = true;
         }
         else {
-            flux_log_error (ctx->h, "Unknown module option: '%s'", argv[i]);
+            flux_log (ctx->h, LOG_ERR, "Unknown module option: '%s'", argv[i]);
```

Comment:
perhaps should stylize do a similar change in `content-files`?  (`content-files` sets errno = EINVAL to make it not as bad)

Reply:
oh and I guess w/ `content-s3` too (given follow up commit to this one)